### ISSUE TITLE
Display spatial and temporal resolution in sidebar

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/ckanext-snap_theme#7]

**Comment by brucecrevensten** [comment]
Spatial resolution is a go, but I needed to patch the CKAN CSW harvester for it to work, by editing `ckanext-spatial/ckanext/spatial/model/harvested_metadata.py` [link to source](https://github.com/ckan/ckanext-spatial/blob/master/ckanext/spatial/model/harvested_metadata.py#L149) and change the `gml` namespace line from:

``` python
"gml": "http://www.opengis.net/gml/3.2",
```

to:

``` python
"gml": "http://www.opengis.net/gml",
```

I've emailed the CKAN dev list about why this is necessary.  Before making this change, the spatial resolution and units didn't come in, and neither did the temporal start/end times.  This may be related to the versions of GeoNetwork and GML that are in play here.
